app/full_text_search.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import re
+
+from google.appengine.api import search as appengine_search
+
+import model
+
+# The index name for full text search
+PERSON_FULLTEXT_INDEX_NAME = 'person_information'
+
+def search(repo, query_obj, max_results):
+    """
+    Searches person with index.
+    Args:
+        repo: The name of repository
+        query_obj: Search word
```

Comment:
"Search query" would be better. If it's "taro yamada", we usually don't call it a word.